src/SFA.DAS.AssessorService.ExternalApiDataSync/Startup/Bootstrapper.cs

```diff
@@ -1,8 +1,12 @@
-﻿using Microsoft.Azure.WebJobs;
+﻿using Microsoft.Azure.Services.AppAuthentication;
+using Microsoft.Azure.WebJobs;
 using Microsoft.Extensions.Logging;
+using SFA.DAS.AssessorService.ExternalApiDataSync.Helpers;
```

Comment:
See [this comment](https://github.com/SkillsFundingAgency/das-assessor-service/pull/1057/files#r517265542) re duplication

Reply:
ExternalApiDataSync has been moved to the Assessor functions repo